README.md

```diff
@@ -113,13 +107,13 @@ config, allowing only specific config to be controlled.
 ### Choosing config to export ###
 
 It is the developer's responsibility to be aware of what configuration should be
-exported to a module, but here is some help! Start by adding items you know you
+exported to a module (or profile), but here is some help! Start by adding items you know you
```

Comment:
Please can we remove this change? Strictly speaking, if we're going to add profiles, we could also add themes. But since profiles are a kind of special case of module, I prefer just 'module'. FYI - that's also why I've referred to them as 'extensions' in code, as that covers all 3 kinds. But I left that out of the Readme, since 'module' is more readily understood than 'extension'.